### PR TITLE
Temperature bug fix

### PR DESCRIPTION
**Pull request by ktkrg**
*Issue #, if available:*
#150 
*Description of changes:*
Adding data to a repeated element needs to be done using add* method instead of set* if we are adding data incrementally.

*Tests:*
Tested on docker.

*Code coverage percentage for this patch:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java [hunk 1]
```diff
@@ -167,6 +167,7 @@ private static void handle(PAThreadException exception) {
     // what to do when the thread has stopped due to an unexpected exception.
     LOG.error("Thread: {} ran into an uncaught exception: {}", exception.getPaThreadName(),
         exception.getInnerThrowable());
+    exception.getInnerThrowable().printStackTrace();
```
Comment (yojs) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java:170]: Should we do this:
LOG.error("Thread: {} ran into an uncaught exception: {}", exception.getPaThreadName(),
        exception.getInnerThrowable(), exception);

to capture the stack trace in the logs ?

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/DummyGraph.java [hunk 2]
```diff
@@ -125,9 +136,9 @@ public void construct() {
     hotNodeClusterRca.addTag(TAG_LOCUS, LOCUS_MASTER_NODE);
     hotNodeClusterRca.addAllUpstreams(Collections.singletonList(hotJVMNodeRca));
 
-    // constructResourceHeatMapGraph();
+    constructResourceHeatMapGraph();
```
Comment (yojs) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/DummyGraph.java:139]: If we are doing this, we might want to add it to muted RCA list. We don't want this enabled by default, right ?

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java (hunk 1 above)
Comment (ktkrg) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java:170]: I'll incorporate this.

**Comment by ktkrg** [comment]
Closing this for a new one.

**Pull request closed without merging**